src/main/java/ch/njol/skript/expressions/ExprSaturation.java

```diff
@@ -19,88 +19,83 @@
  */
 package ch.njol.skript.expressions;
 
-import org.bukkit.entity.HumanEntity;
-import org.bukkit.entity.LivingEntity;
 import org.bukkit.entity.Player;
-import org.bukkit.entity.Projectile;
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
 
 import ch.njol.skript.Skript;
-import ch.njol.skript.bukkitutil.ProjectileUtils;
+import ch.njol.skript.classes.Changer;
 import ch.njol.skript.classes.Converter;
 import ch.njol.skript.classes.Changer.ChangeMode;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
-import ch.njol.skript.expressions.base.PropertyExpression;
 import ch.njol.skript.expressions.base.SimplePropertyExpression;
-import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.ExpressionType;
-import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
 
 /**
  * Used to set saturation of players. Number is used in case something changes in future...
  * @author bensku
+ * 
+ * Cleaned up by LimeGlass (2.2-dev35)
  */
 @Name("Saturation")
-@Description("The saturation of a player.")
-@Examples({"saturation of player is 20 #Not hungry!"})
-@Since("2.2-Fixes-V10")
-public class ExprSaturation extends PropertyExpression<Player, Number> {
+@Description("The saturation of the player(s).")
+@Examples("set saturation of player to 20 #Full hunger")
+@Since("2.2-Fixes-v10, 2.2-dev35 (Converted to SimplePropertyExpression)")
```

Comment:
Again, who cares about a Java thing in Skript?

Reply:
We get it... Just post once.